hokusai/production.yml

```diff
@@ -20,6 +20,8 @@ spec:
       containers:
         - name: kaws-web
           env:
+            - name: PORT
+              value: "4000"
```

Comment:
@eessex Should the port number be in quotes here? if yes, how come?

Reply:
Yes -- we actually encountered this as a breaking change in Positron, and further one that `hokusai staging update --dry-run` doesn't catch. All values declared in the `env` block must be strings.

Reply:
Got it! Thanks